### PR TITLE
1561: Add import store role

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

In preparation of #1475 we need a separate role for importing accepting stores.

### Proposed changes

- add the role PROJECT_STORE_MANAGER
- add runConfig to create a user for nuernberg
- adjust constraint for admin table
- add route with placeholder content
- add navigation and dashboard item
- add backend test for Authorizer

### Side effects

- none

### Resolved issues

Fixes: #1561

### Testing (dev only)
- Migrate DB
- Create a Project Store Manager user
- run backend service
- Switch to Nuernberg and Login
- Check if route `/stores` can be accessed

### CODE REVIEW

File: administration/src/bp-modules/NavigationBar.tsx [hunk 1]
```diff
@@ -67,6 +67,11 @@ const Navigation = (props: Props) => {
             <Button minimal icon='path-search' text='Region verwalten' />
           </NavLink>
         ) : null}
+        {role === Role.ProjectStoreManager ? (
```
Comment (steffenkleinle) [inline administration/src/bp-modules/NavigationBar.tsx:70]: :question: Shouldn't this be available to at least project admins as well? Applies to later usages as well.

:question: Is there documentation somewhere about the different roles and their permissions?

File: administration/src/bp-modules/home/HomeController.tsx [hunk 2]
```diff
@@ -55,6 +55,11 @@ const HomeController = () => {
           <StyledButton icon='path-search' text='Region verwalten' />
         </NavLink>
       ) : null}
+      {role === Role.ProjectStoreManager ? (
```
Comment (steffenkleinle) [inline administration/src/bp-modules/home/HomeController.tsx:58]: Same as above.

File: administration/src/bp-modules/users/UsersTable.tsx [hunk 3]
```diff
@@ -120,6 +120,8 @@ export const roleToText = (role: Role): string => {
       return 'Keine'
     case Role.ProjectAdmin:
       return 'Administrator'
+    case Role.ProjectStoreManager:
+      return 'Akzeptanzpartnerverwalter'
```
Comment (steffenkleinle) [inline administration/src/bp-modules/users/UsersTable.tsx:124]: :upside_down_face: 
```suggestion
      return 'Verwaltung Akzeptanzpartner'
```

File: administration/src/Router.tsx [hunk 4]
```diff
@@ -18,6 +18,7 @@ import RegionsController from './bp-modules/regions/RegionController'
 import DataPrivacyController from './bp-modules/regions/data-privacy-policy/DataPrivacyController'
 import DataPrivacyPolicy from './bp-modules/regions/data-privacy-policy/DataPrivacyPolicy'
 import StatisticsController from './bp-modules/statistics/StatisticsController'
+import StoresController from './bp-modules/stores/StoresController'
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply administration/src/Router.tsx:21]: blueprint. its a library only for desktop usage

File: administration/src/bp-modules/NavigationBar.tsx (hunk 1 above)
Comment (f1sh1918) [reply administration/src/bp-modules/NavigationBar.tsx:70]: its in the backend Role class
no at the moment this is only for our usage as explained in the description so no one should be able to create this role for now except via backend command line

File: administration/src/Router.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply administration/src/Router.tsx:21]: Thank you!

File: administration/src/bp-modules/NavigationBar.tsx (hunk 1 above)
Comment (steffenkleinle) [reply administration/src/bp-modules/NavigationBar.tsx:70]: Thanks for explaining :)

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox, works as expected.

**Pull request merged**